Calmspace/DetailsTableViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  DetailsTableViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 12/10/2023.
+//
+
+import UIKit
+import Foundation
+
+struct PracticeDetail {
+    let title: String
+    let url: URL
+}
+
```

Comment:
In order to make the code look tidy, do not leave multiple empty lines. Leave a single empty line here and everywhere else in the PR.